.github/workflows/php.yml

```diff
@@ -44,6 +44,8 @@ jobs:
       run: composer install --prefer-dist --no-progress --no-suggest
 
     - name: Check Style
+      env:
+        PHP_CS_FIXER_IGNORE_ENV: 1
```

Comment:
I had to add this to bypass the environment check for PHP8 based on [this comment](https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4702#issuecomment-738086179)

I'm planning on removing this enviornment variable once php-cs-fixer is more generally available for PHP8.